object_storage_api/repositories/image.py

```diff
@@ -56,3 +56,27 @@ def get(self, image_id: str, session: ClientSession = None) -> Optional[ImageOut
         if image:
             return ImageOut(**image)
         return None
+
+    def list(self, entity_id: Optional[str], primary: Optional[bool], session: ClientSession = None) -> list[ImageOut]:
+        """
+        Retrieve Images from a MongoDB database.
+
+        :param session: PyMongo ClientSession to use for database operations.
+        :return: List of Images or an empty list if no Images are retrieved
```

Comment:
```suggestion
        :return: List of images or an empty list if no images are retrieved.
```
Me being picky, just we changed these in IMS recently (as well as adding full stops to many comments - although mostly in the tests there).

Also the `entity_id` and `primary` are missing their comments.

Reply:
Is the text in comments meant to be normal grammar (i.e no mid-sentence capitalisation unless a proper noun, acronym, etc)?

In that case should `Retrieve Images from a MongoDB database.` , be changed to `Retrieve images from a MongoDB database.`>

Reply:
Yes like that. When I added systems on IMS all my comments referred to them as `System`, when refactoring all of the tests I realised Viktor's style always used the lower case with full stops so I matched his and used the lower case variant instead `system`. As I said it was a picky point to match what happened on IMS. (Not that the code there actually got changed, just the tests)